platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/configurations/DefaultConfigurationContainer.java

```diff
@@ -27,9 +27,9 @@
 import org.gradle.api.artifacts.ConsumableConfiguration;
 import org.gradle.api.artifacts.Dependency;
 import org.gradle.api.artifacts.DependencyScopeConfiguration;
+import org.gradle.api.artifacts.LegacyConfiguration;
```

Comment:
💅 I get it's re-formatting, but seems very very unrelated to the PR

Reply:
I agree, have no idea why this was formatted. Reverted!